tests/check_baton.c

```diff
@@ -2742,6 +2746,54 @@ START_TEST(test_regression_github_issue242) {
                          json_string("d41d8cd98f00b204e9800998ecf8427e")));
 }
 
```

Comment:
`END_TEST` missing here?

Reply:
Yes, thanks. Works without it, unexpectedly.